cogs5e/models/character.py

```diff
@@ -374,6 +380,8 @@ async def commit(self, ctx, do_live_integrations=True):
 
     async def set_active(self, ctx):
         """Sets the character as globally active and unsets any server-active character or channel-active character in the current context, whichever is most specific."""
+        owner_id = str(ctx.author.id)
+        did_unset_active_location = False
```

Comment:
Looks like these variables don't get used in the function?

Reply:
They do on line 416 & 419

Reply:
Ah, now I see... This was due to merge. My local git I was responding to comments from didn't have the merge done yet.